app/services/action_queue.rb

```diff
@@ -148,7 +149,8 @@ def transaction_payload
             currency:     data[:currency]
           }.merge(fake_card_info),
           action: {
-            source: data[:source]
+            source: data[:source],
+            skip_confirmation: 1
```

Comment:
Ahh okay, that makes perfect sense. Thanks for the explanation. @osahyoun your specs failed on CircleCi.